wger/utils/middleware.py

```diff
@@ -119,3 +122,11 @@ def process_response(self, request, response):
             response['X-wger-redirect'] = request.path
             response.content = request.path
         return response
+
+
+class SocialAuthExceptionMiddleware(SocialAuthExceptionMiddleware):
+    def process_exception(self, request, exception):
```

Comment:
The process_exception() method is an in-built function in SocialAuthExceptionMiddleware which is been overwritten by our custom function. Any method calling the function will be passing the request and the exception arguments but if a request is not passed as argument, a positional argument exception will be thrown.
https://github.com/python-social-auth/social-app-django/blob/master/social_django/middleware.py